src/Arcus.WebApi.Logging/RequestTrackingMiddleware.cs

```diff
@@ -210,7 +211,26 @@ private void LogRequest(string requestBody, string responseBody, HttpContext htt
             Dictionary<string, object> telemetryContext = CreateTelemetryContext(requestBody, responseBody, httpContext.Request.Headers, _logger);
             telemetryContext.Add("Body", "Request body is now available in 'RequestBody' dimension");
 
-            _logger.LogRequest(httpContext.Request, httpContext.Response, duration, telemetryContext);
+            var operationName = BuildOperationName(httpContext);
+
+            _logger.LogRequest(httpContext.Request, httpContext.Response, operationName, duration, telemetryContext);
+        }
+
+        private static string BuildOperationName(HttpContext httpContext)
+        {
+            string operationName = null;
+
+            if (httpContext.Features.Get<Microsoft.AspNetCore.Http.Features.IEndpointFeature>()?.Endpoint is RouteEndpoint re)
+            {
+                operationName = re.RoutePattern.RawText;
+            }
```

Comment:
```suggestion
            if (httpContext.Features.Get<Microsoft.AspNetCore.Http.Features.IEndpointFeature>()?.Endpoint is RouteEndpoint endpoint)
            {
                operationName = endpoint.RoutePattern.RawText;
            }
```

We usually only use abbreviations in lambda's.

Reply:
Fixed this.